src/main/java/com/esri/samples/scene/animate_3d_graphic/README.md

```diff
@@ -1,25 +1,25 @@
-<h1>Animate 3d Symbols</h1>
+<h1>Animate 3d Graphic</h1>
 
-<p>Demonstrates how to animate a graphic's position and rotation and follow it with a camera. Also shows how to combine
-  a SceneView and MapView in an MVC application with property binding.</p>
+<p>Demonstrates how to animate a graphic's position and rotation and follow using an OrbitGeoElementCameraController. 
+Also shows how to combine a SceneView and MapView in an MVC application with property binding.</p>
```

Comment:
Could change this to be worded:
```
This application demonstrates the following:
  - How to animate a graphic's position and orientation
  - Following a graphic using an OrbitGeoElementCameraController
  - Synchronising a SceneView and Mapview in an MVC application with a property binding
```

Or something like that which reads better then a sentence with "and and and".